react/features/welcome/components/WelcomePageLists.js

```diff
@@ -98,14 +127,18 @@ class WelcomePageLists extends Component<Props> {
  * @param {Object} state - The redux state.
  * @protected
  * @returns {{
- *     _hasCalendarEntries: boolean
+ *     _hasRecentListEntries: boolean
  * }}
  */
 function _mapStateToProps(state: Object) {
-    const { events } = state['features/calendar-sync'];
+    const { defaultPage } = state['features/welcome'];
+    const recentList = state['features/recent-list'];
+    const _hasRecentListEntries = Boolean(recentList && recentList.length);
```

Comment:
The underscore in the name is supposed to convey that the property is not public (it's private, protected, internal).